libraries/AP_GPS/AP_GPS_NMEA.h

```diff
@@ -257,6 +270,52 @@ class AP_GPS_NMEA : public AP_GPS_Backend
     } _uniheadinga;
 #endif
 #endif // AP_GPS_NMEA_UNICORE_ENABLED
+
+#if AP_GPS_NMEA_QUECTEL_ENABLED
+    /*
+      Quectel PQTM message parsing structures
+     */
+    // PQTMVER parsing
+    struct {
+        char version[30];        // Version string (e.g., "LG290P03AANR01A05S")
+        char build_date[20];     // Build date (e.g., "2025/05/08")
+        char build_time[20];     // Build time (e.g., "16:23:00")
+    } _pqtmver;
+    bool _have_pqtmver;
+
+    // PQTMEPE parsing - Estimated Position Error
+    struct {
+        float epe_north;
+        float epe_east;
+        float epe_down;
+        float epe_2d;
+        float epe_3d;
+    } _pqtmepe;
+
+    // PQTMVEL parsing - Velocity Information
+    struct {
+        uint32_t time_ms;
+        Vector3f vel_NED;
+        float ground_speed;
+        float speed_3d;
+        float heading;
+        float ground_speed_acc;
+        float speed_acc;
+        float heading_acc;
+    } _pqtmvel;
+
+    // PQTMPVT parsing - PVT Result
+    struct {
+        uint32_t tow;           // Time of week
+        int32_t leap_seconds;   // Leap seconds
+        float sep;              // Geoidal separation (undulation)
+        float pdop;             // PDOP (used for VDOP)
+    } _pqtmpvt;
+
+    uint32_t _last_PQTM_vel_ms;
+    uint32_t _last_PQTM_acc_ms;
+#endif  // AP_GPS_NMEA_QUECTEL_ENABLED
+    bool _expect_pqtm;
```

Comment:
```suggestion
    bool _expect_pqtm;
#endif  // AP_GPS_NMEA_QUECTEL_ENABLED
```

this should be no-compiler-output change if the feature is disabled...